.changeset/rotten-pandas-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": minor
+---
+
+Create multiple Routes based on unique key combinations in of the ContentScope Objects, to enable custom content Scopes
```

Comment:
```suggestion
Support multiple paths `ContentScopeProvider` by default

Change the default implementation of `location.createPath` to create multiple paths based on the content scope shapes.
```

Reply:
changed the description to your suggestion 👍